### PR TITLE
feat(demo): Add Airbnb's review email template

### PR DESCRIPTION
**Pull request by joaom00**

### CODE REVIEW

File: apps/demo/emails/airbnb-review.tsx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+          <br />
+          <Text style={paragraph}>
+            Now that the review period is over, we’ve posted Alex’s review to
+            your Airbnb profile.
+          </Text>
+          <Text style={paragraph}>
+            While it’s too late to write a review of your own, you can send your
+            feedback to Alex using your Airbnb message thread.
+          </Text>
+          <div style={{ padding: '16px 0 24px 0' }}>
```
Comment (bukinoshita) [inline apps/demo/emails/airbnb-review.tsx:50]: Can we not depend on a `div` here?

Comment (joaom00) [reply apps/demo/emails/airbnb-review.tsx:50]: Could be `<Section />` here? or something else?

Comment (bukinoshita) [reply apps/demo/emails/airbnb-review.tsx:50]: Yeah, it could!

**Comment by zenorocha** [comment]
This is live now! Thanks again @joaom00 ;)

https://react.email/examples
https://demo.react.email/preview/airbnb-review

**Pull request merged**